example/v2/example.dart

```diff
@@ -0,0 +1,54 @@
+//import 'dart:convert';
+import 'dart:async';
+//import 'package:enough_convert/enough_convert.dart';
+import 'package:postgres/postgres.dart';
+
+void main(List<String> args) {
```

Comment:
This example file (or the v3 one) won't get picked up by `pub.dev`. We will need an `example/README.md` and link these from there.

Reply:
removed commented codes

Reply:
README.md created